feat(core): nx list enhancements and cleanup (#2676)
nrwl/nx: docs/angular/getting-started/getting-started.md

This will list all the capabilities in the `@nrwl/web` collection.

> `nx list` will also output a list of Nrwl core and community plugins that you may want to consider adding to your workspace.

Reviewer: Reworded so that it doesn't look like we're saying the community plugins are ours.

```suggestion
Alongside the Nrwl core plugins `nx list` will also display some community plugins that you may want to consider adding to your workspace.
```
